Ginger/GingerCore/Drivers/WebServicesDriverLib/WebServicesDriver.cs

```diff
@@ -112,7 +112,12 @@ public class WebServicesDriver : DriverBase
         [UserConfiguredDefault("")]
         [UserConfiguredDescription("Proxy Settings | Host:Port Example: genproxy.amdocs.com:8080")]
         public string WebServicesProxy { get; set; }
-        
+
+        [UserConfigured]
+        [UserConfiguredDefault("false")]
+        [UserConfiguredDescription("Use Proxy Server Settings | Set to true in order to use local Proxy Server settings, if set to true above proxy will not be considered")]
+        public bool UseProxyServerSettings { get; set; }
+
```

Comment:
1- English is wrong here, it supposed to be UseServerProxySettings - please change in field and description
2- dont write "above proxy will not be considered"  because maybe order will be change in some time, change it to "  configured Agent 'Proxy Settings' will be avoided.   "